addons/ofxAssimp/src/Source/ofxAssimpSrcBone.cpp

```diff
@@ -24,99 +24,6 @@ void SrcBone::setAiBone(aiBone* aAiBone, aiNode* aAiNode) {
-////	}
-//	// update //
-////	mAiMatrix = mAiBone->mOffsetMatrix;
-//	mAiMatrix = mAiNode->mTransformation;
-//
-////	mAiMatrixGlobal = mAiNode->mTransformation;
-////	mBoneLocalTransform = glmMat4ToAiMatrix4x4(mAiMatrix);
-//	// and now append all node transformations down the parent chain until we're back at mesh coordinates again
-////	if( bRoot ) {
-////		const aiNode* tempNode = mAiNode;
-////		aiNode* node = scene->mRootNode->FindNode(mAiBone->mName);
-////		const aiNode* tempNode = mAiNode->mParent;
-////		while(tempNode) {
-//////			aiMatrix4x4 m = tempNode->mTransformation;
-//////			m.Transpose();
-////		// check your matrix multiplication order here!!!
-//////			mAiMatrix = m * mAiMatrix;//boneMatrices[a];
-//////			ofLogNotice("SrcBone name: ") << getName() << " parent: " << tempNode->mName.data;
-//////			mAiMatrixGlobal = tempNode->mTransformation * mAiMatrixGlobal;
-////			mAiMatrixGlobal = tempNode->mTransformation * mAiMatrixGlobal;
-////														 // boneMatrices[a] = boneMatrices[a] * tempNode->mTransformation;
-//////			mAiMatrix = tempNode->mTransformation * mAiMatrix;
-//////			mAiMatrix = mAiMatrix * tempNode->mTransformation;
-////			tempNode = tempNode->mParent;
-////		}
-////		mAiMatrix = aGlobalInv * mAiMatrix;
-////	} else {
-////		mAiMatrix = mAiNode->mTransformation * mAiMatrix;
-////		mAiMatrix = mAiMatrix * mAiNode->mTransformation;
-////	}
-//	if( bRoot) {
-////		mAiMatrix = mAiMatrixGlobal;
-//	}
-//
-////	mAiMatrix *= (mOffsetMatrix);
-//
-////	aiVector3t<float> tAiScale;
-////	aiQuaterniont<float> tAiRotation;
-////	aiVector3t<float> tAiPosition;
-////
-////	// this is the local matrix
-////	mAiMatrix.Decompose( tAiScale, tAiRotation, tAiPosition );
-////
-////	glm::vec3 tpos = glm::vec3( tAiPosition.x, tAiPosition.y, tAiPosition.z );
-////	glm::quat tquat = glm::quat(tAiRotation.w, tAiRotation.x, tAiRotation.y, tAiRotation.z);
-//////	glm::quat tquat = glm::quat(tAiRotation.x, tAiRotation.y, tAiRotation.z, tAiRotation.w);
-////	glm::vec3 tscale = glm::vec3( tAiScale.x, tAiScale.y, tAiScale.z );
-////
-//////	if( bRoot ) {
-//////		std::cout << "Bone: " << getName() << " tscale: " << tscale << " | " << ofGetFrameNum() << std::endl;
-//////	}
-////
-//////	if(bRoot) {
-//////		setGlobalPosition( tpos );
-//////		setGlobalOrientation( tquat );
-//////		setScale( tscale );
-//////	} else {
-////		setPositionOrientationScale( tpos, tquat, tscale );
-////	setPosition(tpos);
-////	setOrientation(tquat);
-////	setScale(tscale);
-//
-////	setGlobalPosition(tpos);
-////	setGlobalOrientation(tquat);
-////	setScale(tscale);
-//
-////	mPos = getGlobalPosition();
-////	}
-//
-////	for(auto it = childBones.begin(); it != childBones.end(); ++it ) {
-////		it->second->update();
-////	}
-////	for( auto& kid : childBones ) {
-////		kid->update();
-////	}
-//
-////	getGlobalTransformMatrix();
-//}
-
```

Comment:
should all this be commented out?

Reply:
I think so, it was already commented out, I just removed it. I tested with the examples and they work as expected.
https://github.com/openframeworks/openFrameworks/blob/b4118a0c86a1bf5989f24216e4766efcdd8f3c4f/addons/ofxAssimp/src/Source/ofxAssimpSrcBone.cpp#L27-L118

Reply:
oops - sorry I misread the diff - haha!